src/java/boa/aggregators/ml/LinearRegressionAggregator.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2014, Hridesh Rajan, Robert Dyer,
```

Comment:
wrong year, missing some authors

Reply:
I have fixed the year. I was wondering which authors did I miss?